Updated Readme, Added Total Sizes per Year #3

kmahesh18 opened the pull request.

vanga commented on README.md:
```diff
@@ -21,6 +21,92 @@ PDF judgmetns and the metadata can be downloaded from [Kaggle](https://www.kaggl
+
+**Total Size Across All Years: 52.244 GB**
+
+---
```
Is the total size 50GB? does this include regional language judgments also. I am not entirely sure, but I remember the total dataset size to be higher than this number.

Also, you can include the script that you might have used in the repo. Might also be a good idea to have a csv file that is either embedded in the readme or pointed to in the readme so that it can be programmatically updated

kmahesh18 replied:
I've Looked into it.
Got Total Size approx the same as before....it's right and includes regional judgements too..
I've alse updated readme linking csv....and running python script would update csv

vanga replied:
CSV file and the script is enough, we don't need the .md file again with the same info and we can also omit the info from the main readme. The idea of CSV is to have it get updated every few days automatically. Having the info duplicated or in the .md files would mean it would go out of sync. So, that's not something we want to do.

kmahesh18 commented:
Let me know if you want any other changes

vanga commented on calculate_s3_sizes.py:
```diff
@@ -0,0 +1,104 @@
+import boto3
```
Aren't there any index files that we can rely on to get the sizes, do we actually need to list all the objects to get the sizes? 
We can also add the judgment count against each in addition to the sizes.

kmahesh18 replied:
@vanga No, when index files exist - the script can query specific files but in current situation - index files aren't available, listing is the only way to discover what files exist in S3
I've updated again ..this would be the most optimized approach with available data

kmahesh18 replied:
@vanga Sure, I'll be waiting for the changes made by @rnihesh so that i can calculate sizes and make csv according to index files

kmahesh18 commented:
@vanga Hey, I've updated some changes as you mentioned...calculating sizes by zip_size key in index.jsons and also updated workflow...let me know if you want some changes

vanga commented:
> @vanga Hey, I've updated some changes as you mentioned...calculating sizes by zip_size key in index.jsons and also updated workflow...let me know if you want some changes

I think I am still seeing your old code itself, have you pushed the changes to the same branch?
